cmd/thanos/tools_bucket.go

```diff
@@ -203,6 +205,10 @@ func (tbc *bucketWebConfig) registerBucketWebFlag(cmd extkingpin.FlagClause) *bu
 	cmd.Flag("timeout", "Timeout to download metadata from remote storage").Default("5m").DurationVar(&tbc.timeout)
 
 	cmd.Flag("label", "External block label to use as group title").StringVar(&tbc.label)
+
+	cmd.Flag("enable-mark-deletion", "Enable Mark Deletion button in the UI").Default("false").BoolVar(&tbc.enableMarkDeletion)
+
+	cmd.Flag("enable-mark-no-compaction", "Enable Mark No Compaction button in the UI").Default("false").BoolVar(&tbc.enableMarkNoCompaction)
```

Comment:
I prefer to combine the two flags into one. One flag should be good enough

Reply:
Yeah, maybe something like "disable admin operations"? We should probably enable them by default as that's how it was previously

Reply:
so by default you guys want the buttons to be available in the bucket UI instead? with the flag name `--disable-admin-operations`?